Replicator/Inserter.cc

```diff
@@ -191,25 +191,47 @@ namespace litecore { namespace repl {
     // Callback from c4doc_put() that applies a delta, during _insertRevisionsNow()
     C4SliceResult Inserter::applyDeltaCallback(C4Document *c4doc,
                                                C4Slice deltaJSON,
+                                               C4RevisionFlags *revFlags,
                                                C4Error *outError)
     {
         Doc doc = _db->applyDelta(c4doc, deltaJSON, true);
         alloc_slice body = doc.allocedData();
+        Dict root = doc.root().asDict();
+        FLSharedKeys sk = nullptr;
```

Comment:
Can't this be declared closer to line 209?  Where else is it used?

Reply:
My intention is to not get it again if the body has changed. It was used to generate the changed body, and I can use to generate the new Doc.

Reply:
I missed where it was used on line 220, now I see it.